### PR TITLE
Feature/page allow rebased

### PR DESCRIPTION
**Pull request by tiagojsag**

### CODE REVIEW

**Comment by clementprdhomme** [comment]
@tiagojsag Can you give it a last review? I think it's ready to merge.

File: public/form/css/styles.css [hunk 1]
```diff
@@ -341,141 +349,119 @@ body {
...
+  line-height: 1.5;
 }
 
-.reg form input[type="submit"] {
-  width: 98%;
-  position: relative;
-  left: -1%;
+/*FORM STYLES*/
+
+form {
```
Comment (nerik) [inline public/form/css/styles.css]: (code style) I think we should avoid styling html tags

File: public/form/css/styles.css [hunk 2]
```diff
@@ -341,141 +349,119 @@ body {
...
 input:required {
   box-shadow: inset -6px 0px 0px #ef4323;
 }
 
-.tos span input:required {
-  box-shadow: 0px 0px 0px transparent;
+select {
+  cursor: pointer;
+  color: #a9a9a9;
+  background: url("data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTMiIGhlaWdodD0iOSIgdmlld0JveD0iMCAwIDEzIDkiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHRpdGxlPlRyaWFuZ2xlIDEgQ29weTwvdGl0bGU+PHBhdGggZD0iTTEyIDFMNi41IDYuNyAxIDEiIHN0cm9rZS13aWR0aD0iMiIgc3Ryb2tlPSIjNTU1IiBmaWxsPSJub25lIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiLz48L3N2Zz4=") center right 20px no-repeat, #fff;
```
Comment (nerik) [inline public/form/css/styles.css]: Shouldn't this be handled by webpack? There's no way of telling what the "source" svg is.

File: public/form/css/styles.css [hunk 3]
```diff
@@ -484,92 +470,133 @@ section p {
...
+
+@media screen and (min-width: 769px) {
+  .login_buttons a {
+    flex-basis: calc(50% - 7px);
+  }
 }
 
 .login_buttons a.fb {
-  background-color: #3b5998;
+  background: url("data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTEiIGhlaWdodD0iMjIiIHZpZXdCb3g9IjAgMCAxMSAyMiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj48dGl0bGU+U2hhcGU8L3RpdGxlPjxwYXRoIGQ9Ik03LjA4OCAyMkgyLjgzNlYxMUgwVjcuMjFsMi44MzYtLjAwMi0uMDA0LTIuMjMzQzIuODMyIDEuODgyIDMuNjk3IDAgNy40NTQgMGgzLjEyOHYzLjc5Mkg4LjYyN2MtMS40NjQgMC0xLjUzMy41My0xLjUzMyAxLjUxOGwtLjAwNiAxLjg5OGgzLjUxN2wtLjQxNCAzLjc5TDcuMDkyIDExIDcuMDkgMjJ6IiBmaWxsPSIjRkZGIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiLz48L3N2Zz4=") no-repeat, #3b5998;
```
Comment (nerik) [inline public/form/css/styles.css]: same

File: public/form/css/styles.css [hunk 4]
```diff
@@ -484,92 +470,133 @@ section p {
...
...MgMy44NTZjMCAyLjExNCAxLjcgMy44MjIgNC4yMzYgMy44MjIuMTc3IDAgLjM0OC0uMDA0LjUxNi0uMDE2LS4xNjUuMjk4LS4yODMuNjM1LS4yODMuOTgzIDAgLjU4OC4zMzQgMS4wNjQuNzU3IDEuNDU0LS4zMiAwLS42MjkuMDA4LS45NjUuMDA4LTMuMDg4IDAtNS40NjQgMS44NjEtNS40NjQgMy43OSAwIDEuOSAyLjYwNiAzLjA4OCA1LjY5MyAzLjA4OCAzLjUyIDAgNS40NjQtMS44OSA1LjQ2NC0zLjc5IDAtMS41MjMtLjQ3NS0yLjQzNi0xLjk0NC0zLjQxOEM4LjcxIDkuNDQgNy43NDkgOC42MjIgNy43NDkgOC4xNGMwLS41NjQuMTctLjg0MiAxLjA2OC0xLjUwNS45Mi0uNjggMS41Ny0xLjYzNiAxLjU3LTIuNzQ2IDAtMS4zMjMtLjYyMy0yLjYxNC0xLjc5My0zLjAzOGgxLjc2MmwxLjI0NS0uODUuMDAzLS4wMDJ6TTkuNjYyIDEyLjg2OGMuMDQ0LjE3NS4wNjcuMzU2LjA2Ny41NDIgMCAxLjUzNi0xLjA0NSAyLjczNS00LjA0NSAyLjczNS0yLjEzNSAwLTMuNjc2LTEuMjc3LTMuNjc2LTIuODEzIDAtMS41MDUgMS45MTItMi43NTYgNC4wNDYtMi43MzUuNDk4LjAwNS45NjIuMDggMS4zODQuMjEgMS4xNTguNzYzIDEuOTkgMS4xOTIgMi4yMjUgMi4wNjJsLS4wMDEtLjAwMXpNNi4yNDQgNy4xNGMtMS40MzMtLjA0LTIuNzk1LTEuNTE2LTMuMDQyLTMuMjk1LS4yNDctMS43OC43MTQtMy4xNDIgMi4xNDYtMy4xMDIgMS40MzEuMDQgMi43OTQgMS40NjggMy4wNDEgMy4yNDcuMjQ3IDEuNzgtLjcxMyAzLjE4OS0yLjE0NiAzLjE0OWwuMDAxLjAwMXpNMTcuMjcgMy43N1YwSDE1Ljk0djMuNzdoLTMuOTg0djEuMjU1aDMuOTg0djMuNzdoMS4zMjl2LTMuNzdoMy45ODRWMy43N0gxNy4yN3oiIGZpbGw9IiNGRkYiIGZpbGwtcnVsZT0iZXZlbm9kZCIvPjwvc3ZnPg==") no-repeat, #db3236;
```
Comment (nerik) [inline public/form/css/styles.css]: same

File: public/form/css/styles.css [hunk 5]
```diff
@@ -484,92 +470,133 @@ section p {
...
   display: flex;
-  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
 }
 
-.column {
-  width: 48%;
-  display: inline;
+.reg-form div {
```
Comment (nerik) [inline public/form/css/styles.css]: This selector will have very poor performance I think

File: public/form/css/styles.css (hunk 2 above)
Comment (nerik) [inline public/form/css/styles.css]: For posterity: no webpack on this project, but this should definitely be an easily reproducible process

File: public/form/css/styles.css [hunk 6]
```diff
@@ -592,111 +619,329 @@ section.reg form {
...
+.allow .content > ul > li {
+  box-sizing: border-box;
+  display: block;
+}
+
+.allow .content > ul:first-child {
+  margin-right: 0px;
+}
+
+.allow #advise_email {
```
Comment (nerik) [inline public/form/css/styles.css]: Verbose selector (should be #advise_email)

File: public/form/css/styles.css [hunk 7]
```diff
@@ -592,111 +619,329 @@ section.reg form {
...
+
+.link_underline:after {
+  content: "";
+  display: block;
+  height: 1px;
+  background-color: #f04323;
+  margin-top: 1px;
+}
+
+.allow #button_allow {
```
Comment (nerik) [inline public/form/css/styles.css]: same

File: public/form/css/styles.css [hunk 8]
```diff
@@ -592,111 +619,329 @@ section.reg form {
...
+  font-weight: 700;
+  letter-spacing: -0.1px;
+  cursor: pointer;
+  padding: 20px 100px;
+  display: block;
+  margin: 0 auto 30px;
+  text-align: center;
+}
+
+.allow #button_allow:after {
```
Comment (nerik) [inline public/form/css/styles.css]: same

File: public/form/css/styles.css [hunk 9]
```diff
@@ -592,111 +619,329 @@ section.reg form {
...
+  display: none;
+}
+
+.allow #allow_deny {
```
Comment (nerik) [inline public/form/css/styles.css]: same

File: public/form/css/styles.css [hunk 10]
```diff
@@ -592,111 +619,329 @@ section.reg form {
...
+  section p {
+    text-align: center;
   }
 
-  section {
-    justify-content: flex-start;
-    padding: 10% 0 0!important;
-    height: 170%;
-    max-height: 170%;
+  .log span {
```
Comment (nerik) [inline public/form/css/styles.css]: Bad performance selector

File: public/form/css/styles.css [hunk 11]
```diff
@@ -592,111 +619,329 @@ section.reg form {
...
+    transform: translate(-40%, 0px);
   }
 
-  .pwd p {
-    font-size: 35px;
-    line-height: 40px;
+  .dropdown_content li {
+    display: block;
+    height: 40px;
+    padding-left: 0 !important;
```
Comment (nerik) [inline public/form/css/styles.css]: You shouldn't use !important :trollface:

File: public/form/resetpassword.html [hunk 12]
```diff
@@ -72,11 +72,11 @@ <h1 class="logo">Global Fishing Watch</h1>
 	</header>
 	<section class="pwd">
 		<h1>Password reset</h1>
-		<p>Please enter the email address that you used when creating your account.</p>
+		<p id="first_p">Please enter the email address that you used when creating your account.</p>
```
Comment (nerik) [inline public/form/resetpassword.html]: Use :first-child instead ?

File: public/form/css/styles.css (hunk 1 above)
Comment (clementprdhomme) [inline public/form/css/styles.css]: Completely agree. Don't think we'll have time to refactor the CSS though.

**Pull request merged**